src/librustc_trans/Cargo.toml

```diff
@@ -19,8 +20,8 @@ rustc-demangle = "0.1.4"
 rustc = { path = "../librustc" }
 rustc_allocator = { path = "../librustc_allocator" }
 rustc_back = { path = "../librustc_back" }
-rustc_bitflags = { path = "../librustc_bitflags" }
 rustc_const_math = { path = "../librustc_const_math" }
+rustc_cratesio_shim = { path = "../librustc_cratesio_shim" }
```

Comment:
Since this (and a number of other crates) transitively depend on `syntax` you don't need to link this crate in

Reply:
True, but that feels fragile to me. What's the downside to always including this?

Reply:
Let's go ahead and remove this b/c otherwise it's just slightly confusing why it's here.

Reply:
Seems more confusing to me to omit it; as a naive reader I'd expect this to be present anywhere that bitflags is used.

Reply:
Let's omit it. This is just a "hack" and should be as contained as possible.

Reply:
OK. Done.